internal/handlers/msg_comparison_test.go

```diff
@@ -0,0 +1,345 @@
+package handlers
+
+import (
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+	"github.com/stretchr/testify/assert"
+	"math"
+	"testing"
+	"time"
+)
+
+const (
+	DoubleDataType                 = "double"
+	DoubleZeroDataType             = "double-zero"
+	DoubleNegativeInfinityDataType = "double-negative-infinity"
+	DoublePositiveInfinityDataType = "double-positive-infinity"
+	ArrayDataType                  = "array"
+	EmptyArrayDataType             = "array-empty"
+	DoubleNanDataType              = "double-nan"
+	DoubleMaxDataType              = "double-max"
+	DoubleSmallestDataType         = "double-smallest"
+	StringDataType                 = "string"
+	EmptyStringDataType            = "string-empty"
+	DataDataType                   = "data"
+	BoolDataType                   = "bool"
+	DatetimeDataType               = "datetime"
+	Int32DataType                  = "int32"
+	Int32ZeroDataType              = "int32-zero"
+	Int32MaxDataType               = "int32-max"
+	Int32MinDataType               = "int32-min"
+	Int64DataType                  = "int64"
+	Int64ZeroDataType              = "int64-zero"
+	Int64MaxDataType               = "int64-max"
+	Int64MinDataType               = "int64-min"
+	TimestampDataType              = "timestamp"
+	SchemaWithAllTypes             = "values"
+	CollectionWithAllTypes         = "values"
+	BinaryDataType                 = "binary"
+)
```

Comment:
Why we need those constants?

Reply:
I've been thinking it's convenient to set the data type names to constants and then use them as keys to access the data value.
Regarding to schema and collection constants, if the name of schemes or collections changes, it will also be easier to change the values of constants in one place than to run through all files and change manually